zetaclient/chains/solana/signer/signer.go

```diff
@@ -299,8 +315,21 @@ func (signer *Signer) broadcastOutbound(
 				logger.Error().Err(err).Fields(lf).Msgf("unable to deserialize PDA info")
 			} else if pda.Nonce > nonce {
 				logger.Info().Err(err).Fields(lf).Msgf("PDA nonce %d is greater than outbound nonce, stop retrying", pda.Nonce)
+				if nonce > signer.maxProcessedNonce.Load() {
+					signer.maxProcessedNonce.Store(nonce)
+				}
 				break
 			}
+			if signer.maxProcessedNonce.Load() == 0 {
+				signer.maxProcessedNonce.Store(pda.Nonce)
+			}
+		}
+
+		maxProcessedNonce := signer.maxProcessedNonce.Load()
+		if nonce > maxProcessedNonce+1 && maxProcessedNonce != 0 {
```

Comment:
If the increasing backoff is not what you want, you could try `retry.DoWithBackoff` with constant backoff parameter, like [here](https://github.com/zeta-chain/node/blob/solana-lead-relayer/zetaclient/chains/bitcoin/signer/signer.go#L159). Not sure what this refinement is intended for, would be good to put a piece description for easy understanding, same for above ones.